docs/standard/language-independence.md

```diff
@@ -314,1591 +160,197 @@ Non-compliant type | Description | CLS-compliant alternative
 [UInt64](xref:System.UInt64) | 64-bit unsigned integer | [Int64](xref:System.Int64) (may overflow), [BigInteger](xref:System.Numerics.BigInteger), or [Double](xref:System.Double)
 [UIntPtr](xref:System.UIntPtr) | Unsigned pointer or handle | [IntPtr](xref:System.IntPtr)
 
-The .NET Class Library or any other class library may include other types that aren't CLS-compliant; for example:
+The .NET Class Library or any other class library may include other types that aren't CLS-compliant, for example:
 
-* Boxed value types. The following C# example creates a class that has a public property of type `int*` named `Value`. Because an `int*` is a boxed value type, the compiler flags it as non-CLS-compliant.
+- Boxed value types. The following C# example creates a class that has a public property of type `int*` named `Value`. Because an `int*` is a boxed value type, the compiler flags it as non-CLS-compliant.
 
-  ```csharp
-  using System;
+  [!code-csharp[Conceptual.CLSCompliant#26](../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.clscompliant/cs/box2.cs#26)]
 
-  [assembly:CLSCompliant(true)]
+- Typed references, which are special constructs that contain a reference to an object and a reference to a type. Typed references are represented in .NET by the <xref:System.TypedReference> class.
 
-  public unsafe class TestClass
-  {
-     private int* val;
+If a type is not CLS-compliant, you should apply the <xref:System.CLSCompliantAttribute> attribute with an `isCompliant` value of `false` to it. For more information, see [The CLSCompliantAttribute attribute](#the-clscompliantattribute-attribute) section.
 
-     public TestClass(int number)
-     {
-        val = (int*) number;
-     }
+The following example illustrates the problem of CLS compliance in a method signature and in generic type instantiation. It defines an `InvoiceItem` class with a property of type <xref:System.UInt32>, a property of type `Nullable(Of UInt32)`, and a constructor with parameters of type <xref:System.UInt32> and `Nullable(Of UInt32)`. You get four compiler warnings when you try to compile this example.
```

Comment:
Is there a reason for using VB syntax here?

Reply:
Good catch, not sure why it was written that way.